packages/web/src/components/session-menu/table-of-contents-panel.tsx

```diff
@@ -0,0 +1,68 @@
+import { marked, type Tokens } from 'marked';
+import { CodeCellType, MarkdownCellType, TitleCellType } from '@srcbook/shared';
+import { Circle } from 'lucide-react';
+import { cn } from '@/lib/utils';
+import { useCells } from '@/components/use-cell';
+
+type PropsType = Record<string, never>;
+
+export default function SessionMenuPanelTableOfContents(_props: PropsType) {
+  const { cells: allCells } = useCells();
+  const cells = allCells.filter((cell) => {
+    return cell.type === 'title' || cell.type === 'markdown' || cell.type === 'code';
+  }) as Array<TitleCellType | CodeCellType | MarkdownCellType>;
+
+  return (
+    <>
+      <div className="text-lg font-semibold leading-tight mb-4">
+        <span>Table of contents</span>
+      </div>
+
+      <div className="max-w-60 text-tertiary-foreground pr-10">
+        {cells.map((cell) => {
+          const isRunningCell = cell.type === 'code' && cell.status === 'running';
+          return (
+            <div
+              key={cell.id}
+              className={cn(
```

Comment:
The font looks bigger here than the designs. I believe figma has it at text-sm (14px) but I think it's inheriting text-base (16px)